docs/orchestrator.md

```diff
@@ -132,17 +132,33 @@ blobstream orchestrator start --evm.account 0x966e6f22781EF6a6A82BBB4DB3df8E225D
 
 > **_NOTE:_** The above command assumes that the necessary configuration is specified in the  `<orchestrator_home>/config/config.toml` file.
 
-Then, you will be prompted to enter your EVM key passphrase so that the orchestrator can use it to sign attestations. Make sure that it's the EVM address that was provided when creating the validator. If not, then the orchestrator will not sign, and you will keep seeing a "validator not part of valset" warning message. If you see such message, first verify that your validator is part of the active validator set. If so, then probably the EVM address provided to the orchestrator is not the right one, and you should check which EVM address is registered to your validator. Check the next section for more information.
+Then, you will be prompted to enter your EVM key passphrase so that the orchestrator can use it to sign attestations. Make sure that it's the EVM address that was provided when creating the validator. If not, then the orchestrator will not sign, and you will keep seeing a "validator not part of valset" warning message. If you see such message, first verify that your validator is part of the active validator set. If so, then probably the EVM address provided to the orchestrator is not the right one, and you should check which EVM address is registered to your validator. Check the [Register EVM Address](#register-evm-address) section for more information.
 
-If you no longer have access to your EVM address, you could always edit your validator with a new EVM address. This can be done through the `edit-validator` command. Check the next section.
+If you no longer have access to your EVM address, you could always edit your validator with a new EVM address. This can be done through the `edit-validator` command. Check the [Register EVM Address](#register-evm-address) section.
 
 ### Open the P2P port
 
 In order for the signature propagation to be successful, you will need to expose the P2P port, which is by default `30000`.
 
 If not, then the signatures may not be available to the network and relayers will not be able to query them.
 
-#### Register EVM Address
+### Known issues
+
+#### `transport: authentication handshake failed`
+
+```text
+rpc error: code = Unavailable desc = connection error: desc = "transport: authentication handshake failed: tls: first record does not look like a TLS handshake" 
+```
+
+Seeing this error means that the orchestrator/relayer is trying to connect to a gRPC endpoint that is not secure. To bypass this, use the `--grpc.insecure` flag. However, we recommend using secure gRPC connections.
```

Comment:
```suggestion
Seeing this error means that the orchestrator/relayer is trying to connect to a gRPC endpoint that is not secure. To bypass this, use the `--grpc.insecure` flag.
```

in general, the sdk and comet don't because these things are usually behind a local network. This is also why imo we should just use insecure by default

Reply:
> do you know why we are able to query nodes from the cli without setting up tls?

That depends on the queries. If the query requires a gRPC connection, then the TLS is enabled by default. If the request is only targeting RPC, then there is no TLS for that.

You could try it using:
```sh
celestia-appd status
```
Which will query only the RPC. This will return.

However, for something like:
```
celestia-appd query qgb attestation 1413 --node rpc.consensus.celestia-arabica-10.com:26657 --grpc-addr consensus-validator.celestia-arabica-10.com:9090
```

It will require the `--grpc-insecure` flag.

Reply:
thanks for helping me figure this out and thinking about this before hand but tbc this is not blocking this PR (hence the approve)

I'm still of the opinion that the default should work without requiring the use and knowledge of a flag. So either we enable self signing by default on the server side, or by default we enable insecure connections on the client side. Having something that doesn't work by default is stupid imo

> That depends on the queries. If the query requires a gRPC connection, then the TLS is enabled by default. If the request is only targeting RPC, then there is no TLS for that.

yeah I get that part

I've always assumed that this uses grpc under the hood by swapping the port and protocol, then using insecure as a default. In reality I think this is just making an ABCI Query over rest. Either that or the gateway mapping is being used, as this works as expected

```
celestia-appd q qgb attestation 1  --node http://consensus.lunaroasis.net:26657
```

Reply:
Aaah got you. I will merge this then open an issue to switch the default to insecure so it works out of the box. Thanks :+1:

Reply:
https://github.com/celestiaorg/orchestrator-relayer/issues/594